html/videoroomtest.js

```diff
@@ -162,7 +163,14 @@ $(document).ready(function() {
 											myid = msg["id"];
 											mypvtid = msg["private_id"];
 											Janus.log("Successfully joined room " + msg["room"] + " with ID " + myid);
-											publishOwnFeed(true);
+											if(subscriber_mode) {
+                                                						$('#videojoin').hide();
+                                                						$('#videos').removeClass('hide').show();
+                                            						}
+                                            						else {
```

Comment:
Sorry, still missing the `else` on the same line as the closing bracket :hand_over_mouth: